### DISCUSSION TITLE
DateTimePicker - time() not updating dynamically

### DISCUSSION THREAD

**Discussion by cirrion-software** [filamentphp/filament#14670] [Help]
### Package

Form builder

### Package Version

v3.2.121

### How can we help you?

I am dont want to display the time in a dateTimePicker if a Toggle is on. Simplifying my code I have:

`Toggle::make('all_day')->live()`

` DateTimePicker::make('start')->time(fn(Get $get) => !$get('all_day'))->native(false),`

So if the toggle is on (true) the DateTimePicker time(false). This only updates when I manually refresh the page, it is not updating "live()".

Any ideas?

**Comment by leandrocfe** [comment]
try this

```php
DateTimePicker::make('start')
    ->extraAttributes(['wire:key' => str()->random()])
    ...
```
